packages/frontend/src/components/composer/Composer.tsx

```diff
@@ -390,6 +413,15 @@ const Composer = forwardRef<
             />
           </div>
         </div>
+        {showAppPicker && (
+          <OutsideClickHelper onClick={() => setShowAppPicker(false)}>
```

Comment:
This closes the Picker when a click outside happens. That is not nice for accessibility where no mouse might be involved. Not sure what is best practice here. We could also close the picker on "escape" key?

Reply:
> We could also close the picker on "escape" key?

yes